Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:12-alpine
```

Comment:
Node.js 12.x is EOL'ed  I'd change this to 16.x
https://nodejs.dev/en/download/

Reply:
Raised as a ticket #129